### DISCUSSION TITLE
¡Bienvenido a la versión beta de GitHub Discussions!

### DISCUSSION THREAD

**Discussion by Splaktar** [angular-hispano/angular-hispano#160] [General]
¿Has oído hablar de la nueva función de Discussions de GitHub que se [anunció en GitHub Satellite](https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/#discussions) este mes?

Por favor pruébalo! Menciona a tus amigos. Comparte tus ideas. Pedir ayuda.

**Comment by Splaktar** [comment]
@Villanuevand está aquí! 🚀  ¿Qué te parece mi pana? ✨

**Reply by Villanuevand** [reply]
Esto si que esta interesante para opinar y discutir.
Cada comentario es una potencial respuesta.

**Comment by AXEL1988** [comment]
@Splaktar esto esta para darle un vistazo!

**Comment by devpato** [comment]
Hola senor Splaktar!

**Reply by Splaktar** [reply]
Hola 👋🏼

**Comment by AXEL1988** [comment]
Me parece que sería funcional que en esta sección de Discussions también se pueda visualizar el número de Discussions existentes, que opinan?

**Reply by Splaktar** [reply]
En la pestaña?  ¿Solo mostraría discusiones sin respuesta?

**Reply by Villanuevand** [reply]
Si miras la pantalla inicial de Discussions esta esa posibilidad.

Acá ahora estamos en una discusión puntual. Acá muestra el número de respuestas y algo que me interesaría entender que son los "points" o puntos.

**Comment by Villanuevand** [comment]
Una de las cosas que más me agrada es el historial de acciones. Marque un respuesta como solución, posteriormente la desmarque.

**Reply by Villanuevand** [reply]
Esta muy interesante también la posibilidad que el menú de cada respuestas donde puedes:
 - copiar el link de la respuesta 
 - comentar a partir de una respuesta 
 - referencia un nuevo issue a partir del comentario
Y también, 
 - editar 
 - responder 
 - borrar

**Reply by Splaktar** [reply]
También podemos convertir los problemas en discusión si resultan ser más una discusión que un informe de error.
